Vagrantfile

```diff
@@ -77,79 +77,38 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   # You will need to create the manifests directory and a manifest in
   # the file default.pp in the manifests_path directory.
 
-  #config.vm.provision "puppet" do |puppet|
-  #end
-
   config.vm.define "server" do |server|
-      server.vm.provision "puppet" do |puppet|
-          puppet.manifest_file = "server.pp"
-     puppet.environment = "production"
-     puppet.environment_path = "environments"
-     puppet.module_path = "modules"
-     puppet.manifests_path = "manifests"
-     puppet.options = "--verbose --debug"
-      end
-      server.vm.network :private_network, ip: "172.16.0.2", :netmask => "255.255.0.0", :dns => "127.0.0.1"
+    server.vm.provision "puppet" do |puppet|
+      puppet.manifest_file = "server.pp"
+      puppet.environment = "production"
+      puppet.environment_path = "environments"
+      puppet.module_path = "modules"
+      puppet.manifests_path = "manifests"
+    end
+    server.vm.network :private_network, ip: "172.16.0.2", :netmask => "255.255.0.0", :dns => "127.0.0.1"
   end
 
   config.vm.define "agent" do |agent|
-      agent.vm.provision "puppet" do |puppet|
-          puppet.manifest_file = "client.pp"
-     puppet.environment = "production"
-     puppet.environment_path = "environments"
-     puppet.module_path = "modules"
-     puppet.manifests_path = "manifests"
-     puppet.options = "--verbose --debug"
-      end
-      agent.vm.network :private_network, ip: "172.16.0.3", :netmask => "255.255.0.0"
+    agent.vm.provision "puppet" do |puppet|
+      puppet.manifest_file = "client.pp"
+      puppet.environment = "production"
+      puppet.environment_path = "environments"
+      puppet.module_path = "modules"
+      puppet.manifests_path = "manifests"
+      # puppet.options = "--verbose --debug"
```

Comment:
If verbosity is decreased, maybe it should be decreased for the server as well? Just to be consistent.